device/samples/simple_sample_device_x509.js

```diff
@@ -14,13 +14,13 @@ var Message = require('azure-iot-device').Message;
 
 // String containing Hostname and Device Id in the following format:
 //  "HostName=<iothub_host_name>;DeviceId=<device_id>;x509=true"
-var connectionString = '<DEVICE CONNECTION STRING WITH x509=true>';
+var deviceConnectionString = process.env.DEVICE_CONNECTION_STRING;
 var certFile = '<PATH-TO-CERTIFICATE-FILE>';
 var keyFile = '<PATH-TO-KEY-FILE>';
 var passphrase = '<KEY PASSPHRASE IF ANY>';
```

Comment:
One could ask the question should there be any of these other items that should be replaced with an environment variable.
passphrase?

Also, for x509 connection strings, the are NO secrets in that connection string.  Should it have been made an environment variable then?